Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -7288,7 +7296,7 @@ def _http_request(self, method, url_suffix='', full_url=None, headers=None, auth
                     files=files,
                     headers=headers,
                     auth=auth,
-                    timeout=timeout,
+                    timeout=(10, timeout),
```

Comment:
I would leave this as it was instead of hardcoding the 10. It will lead to edge cases.